distribution/docs/src/main/resources/_contents/_core-concepts/metadata-intro-contents.adoc

```diff
@@ -0,0 +1,56 @@
+
+== Understanding Metadata and Metacards
+
+Metadata is information about a resource, organized into a schema to make it possible to search against.
+The ${ddf-catalog} stores this metadata and allows access to it.
+Metacards are single instances of metadata, representing a single record, in the Metadata Catalog (MDC).
```

Comment:
I'd like to try and be consistent in our terminology. In my opinion, a `Metacard` is the DDF implementation of representing *metadata* about a *resource* (would also be open to *product* but we should pick one and stick with it). I think *record* can be confusing because it could refer to either the metadata or the resource.